File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/While/Exercício 07.py
#Exercício - Pares Consecutivos

soma = 0

x = int(input("Digite um número inteiro: "))

while x != 0:

    if x % 2 != 0:
        x = x + 1

    soma = 5 * x + 20

    print("Soma = ", soma)

    x = int(input("Digite um número inteiro: "))

File: 06 - C/Exercícios/03 - Estruturas Repetitivas/While/Exercício 07.c
//Exercício - Pares Consecutivos

#include <stdio.h>

int main() {

    int x, soma;

    soma = 0;

    printf("Digite um numero inteiro: ");
    scanf("%d", &x);

    while(x != 0){
        
        if(x % 2 != 0){
            x = x + 1;
        }

        soma = 5 * x + 20;

        printf("Soma = %d\n", soma);

        printf("Digite um numero inteiro: ");
        scanf("%d", &x);
    
    }

    return 0;

}